### ISSUE TITLE
Object "shaking" or "vibrating" in the wind.

### ISSUE THREAD

**Issue by not-shady** [lucasvinbr/blender_io_GTA5Ped#5]
I exported a bandana and when I see it in the server it is shaking back and forth rapidly like it is being blown by the wind. When I go inside a building the shaking stops, do you know what could be causing this?
Here is a clip:
https://medal.tv/games/gta-v/clips/V6mMKYMgaBCda/d13370UJg3F9?invite=cr-MSxpZVcsNDU5NDAyNyw

**Comment by lucasvinbr** [comment]
Hi! Wow, I've never seen that before.
You've copied the weights and skeleton from another model, right?
Maybe it could be a shader configuration in the bandana's ODR file?

**Comment by 3941** [comment]
It's definitely a shader, yeah. I think you can control it by vertex painting, but I don't know which channel, or if Blender can even do that (or if your plugin can export it... I know it from 3DSMax and whatever the openformats plugin for that one was called). It should be possible to simply use another shader, though.

**Comment by lucasvinbr** [comment]
The vertex painting theory makes sense... this plugin always exports what I assume to be the vertex color values with all 255's

**Comment by 3941** [comment]
Yeah that'd make perfect sense, actually. I guess the bandana originally had the shader only slightly applied, to simulate some slight reaction to wind. When applied 100% to all vertexes, the whole thing seems to be shifting.

**Comment by not-shady** [comment]
Oh wow yeah y'all are probably right, I am fairly new to this stuff any idea what I need to fix it? Do I need to change any of the values in the otd's or the mesh file to remove the shader or is that something I would need to do in like zmodeler or something?

**Comment by lucasvinbr** [comment]
The file you probably want to check is the .odr file, which should have info on the shaders (and their settings) used on the bandana model. 
If possible, could you please share the contents of the file as well? I'm curious about the parameters that make it shake like that hahaha

**Comment by not-shady** [comment]
Yeah thats what I meant odr lol smh. And yes I will upload the item to here in a bit so you can have a look. I have to resize it again.

**Comment by not-shady** [comment]
[0xDCFF8B64.txt](https://github.com/lucasvinbr/blender_io_GTA5Ped/files/10761722/0xDCFF8B64.txt)

**Comment by not-shady** [comment]
this is the odr file for hair that I have that is doing the same shaking, I am defeated I can not figure out how to get it to stop lol.

**Comment by 3941** [comment]
Does the bandana also use the hair shader?
I've exported hair before and I don't think it was shaking.

**Comment by not-shady** [comment]
no they use different shaders, the bandana was ped.sps

**Comment by 3941** [comment]
Hmm, then maybe any shader can shake, and the magic is in the vertex data.
But that means everything should be shaking...
Unless there's indeed something in the models that gets imported into Blender AND exported out of it....

**Comment by not-shady** [comment]
Yeah, I feel like ive tried everything. any suggestions to try?

**Comment by 3941** [comment]
Not really, but maybe you could upload the whole project? Just zip it and use maybe google drive or whatever file hoster you like. Most people have a google account though. If you do, you'll find your drive at drive.google.com . Just upload there and create a sharable link. Or maybe upload it here... I don't know the github size limits, though.

I'd look into it, compare some of my own files. And maybe we can find something that @lucasvinbr can fix in the plugin, or something we can at least fix manually in the files.
I dunno. No promises though.
